### PR TITLE
bugfix button/about

### PR DESCRIPTION
**Pull request by Alisa-Bogatyryova**

### CODE REVIEW

**Review by Squidoo22 — DISMISSED** [review]

File: src/sass/layouts/_about.scss [hunk 1]
```diff
@@ -171,13 +171,18 @@
   }
 }
 .about__btn {
-  max-width: 176px;
+  display: flex;
+  width: 176px;
```
Comment (izachok) [inline src/sass/layouts/_about.scss]: change to min-width. In this case button with possible longer text will look also good

File: src/sass/layouts/_about.scss [hunk 2]
```diff
@@ -171,13 +171,18 @@
...
   height: 44px;
   justify-content: space-between;
+  font-size: 14px;
+  @include for-tablet-up {
+    font-size: 16px;
+  }
 }
 .btn__arrow {
+  display: block;
   fill: var(--secondary-btn-background-color);
```
Comment (izachok) [inline src/sass/layouts/_about.scss:185]: add margin-left 13px for arrow

**Review by izachok — CHANGES_REQUESTED** [review]

File: src/sass/layouts/_about.scss (hunk 2 above)
Comment (Alisa-Bogatyryova) [reply src/sass/layouts/_about.scss:185]: done

File: src/sass/layouts/_about.scss (hunk 1 above)
Comment (Alisa-Bogatyryova) [reply src/sass/layouts/_about.scss]: done

**Review by izachok — APPROVED** [review]

**Pull request merged**